homeassistant/components/evohome/entity.py

```diff
@@ -101,6 +100,8 @@ def __init__(
         """Initialize an evohome-compatible child entity (DHW, zone)."""
         super().__init__(evo_broker, evo_device)
 
+        self._evo_tcs = evo_device.tcs
```

Comment:
Only used in `EvoZone` subclass, should it be set there instead? (I did check that `tcs` is available on both `evo.HotWater` and `evo.Zone`, as it previously was retrieved from `EvoBroker`.)

Reply:
DHW doesn't use the **tcs**, but it will do so in the future.

This PR moves that attr from all evohome entities (controller/TCS, DHW & zones), and leaves it only with child entities (DHW & zones), so is somewhat of an improvement...

I'd like to leave it in.

Reply:
Sounds good to me!